easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'motif'
+version = '2.3.8'
+
+homepage = 'http://motif.ics.com/'
+description = """Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building
+ applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems.
+ It was the standard toolkit for the Common Desktop Environment and thus for Unix."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['859b723666eeac7df018209d66045c9853b50b4218cecadb794e2359619ebce7']
+
+dependencies = [
+    ('X11', '20171023'),
```

Comment:
@SimonPinches For several of these dependencies we should use more recent versions, and then try to stick to those for the `2018` 'cycle'.

Cfr. #5810 for X11, we should definitely look into updating `libpng` and `freetype` too for use in easyconfigs using a `2018a` toolchain.

Reply:
I guess this will need to wait until a new X11 version is agreed upon?

Reply:
#5810 is merged now, so you can update this to `20180131`